kodilla-collections/src/main/java/com/kodilla/collections/sets/homework/Stamp.java

```diff
@@ -22,7 +23,10 @@ public String getStampName() {
 
     public double getStampSize() {
         return stampWidth * stampHeight;
+    }
```

Comment:
ta metoda nie liczy size :) przy okazji, po co ci pole stampSize, skoro z niego nie korzystasz? 
Ja w tej metodzie zwróciłbym Stringa: return String.format("%.0 x %.0", stampWidth, stampHeight);

Metoda drukująca, czy znaczek ostemplowany czy nie:
public String getStamped() {
return stamped ? "stamped" : "not stamped";
}